README.md

```diff
@@ -0,0 +1,132 @@
+# TABObserverSet
+
+`TABObserverSet`, originally conceived by [Mike Ash](https://github.com/mikeash/SwiftObserverSet), provides a Swift-y alternative to the traditional `NotificationCenter` style of reactive programming.
+
+With a simple syntax, `TABObserverSet` is easy to use and read in your code.
```

Comment:
I'll add a placeholder, currently not available as a pod but will be in the near future